examples/rust/Earthfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.24.0
+
+WORKDIR /usr/src/app
+
+build:
+    COPY . .
```

Comment:
We generally copy files one by one in the examples to make it clearer for the reader.